ambari-infra-manager-it/pom.xml

```diff
@@ -36,6 +36,9 @@
     <failsafe-plugin.version>2.20</failsafe-plugin.version>
     <infra-manager.docker.host>localhost</infra-manager.docker.host>
     <stories.location>NONE</stories.location>
+    <okhttp-version>2.7.5</okhttp-version>
+    <generated-sources-java-path>main/java</generated-sources-java-path>
```

Comment:
i think you can use ${project.basedir} here, probably that is why the build fails

Reply:
this property value is a relative path it is used with
```<generatedSourcesPath>${project.build.directory}/generated-sources</generatedSourcesPath>```